source/includes/connect/client.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$client = new MongoDB\Client("mongodb://localhost:27017");
```

Comment:
Missing newline at end of file.
```suggestion
$client = new MongoDB\Client("mongodb://localhost:27017");

```